src/systemd/sd-dhcp-protocol.h

```diff
@@ -0,0 +1,205 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#ifndef foosddhcpprotocolhfoo
+#define foosddhcpprotocolhfoo
+
+/***
+  Copyright © 2013 Intel Corporation. All rights reserved.
```

Comment:
Is this copyright line right? Where is it coming from?

Reply:
It was mistakenly included when I used sd-dhcp-client.h as a template, I will drop it.

Reply:
ok, same in the other file